supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -78,6 +78,19 @@
 
 *See also*: xref:distinguished-name[distinguished name]
 
+[discrete]
+[[basic-http-authentication]]
+==== image:images/yes.png[yes] Basic HTTP authentication (noun)
```

Comment:
Should we add another entry for "Basic authentication", with each entry having a "See also" link to the other, similar to the "AMQ" and "Red Hat AMQ" entries? Or would that be overkill when the entries are sequential?

Maybe instead include both entries in the term, similar to the "asap, ASAP" entry in the _IBM Style_ word usage topic? 

In either of the above cases, we should list them alphabetically by letter (ignoring case): "Basic authentication, Basic HTTP authentication" -- this approach has the advantage of listing the most likely search term first?

Reply:
Just to clarify, the above comments are for the term only, not the term definition -- I agree with the current Description text which mentions the longer term first  :)

Reply:
Just realised that if we have two entries, they won't actually be sequential -- the "basically" entry would separate them :)

Reply:
I think it's fine to keep it simple and leave it as-is. But I'd like to hear what others think about the suggestion and am open to changing it if that's the consensus.

Reply:
I don't have a strong opinion here. I think we do sometimes have both as entries, but not always. I'm okay with just the one entry.

Reply:
To keep things simple, I'm leaving this as a single entry.